coral-common/build.gradle

```diff
@@ -16,4 +16,12 @@ dependencies {
   }
 
   compile deps.'hadoop'.'hadoop-common'
+
+  // LinkedIn Iceberg dependencies
```

Comment:
Another reason why this needs to be a plugin. This should integrate with OSS Iceberg too.